stable/traefik/README.md

```diff
@@ -88,7 +88,8 @@ The following tables lists the configurable parameters of the Traefik chart and
 | `image`                         | Traefik image name                                                   | `traefik`                                 |
 | `imageTag`                      | The version of the official Traefik image to use                     | `1.3.1`                                  |
 | `serviceType`                   | A valid Kubernetes service type                                      | `LoadBalancer`                            |
-| `loadBalancerIP`                | An available static IP you have reserved on your cloud platform      | None                                      |
+| `loadBalancer.IP`               | An available static IP you have reserved on your cloud platform      | None                                      |
```

Comment:
No problem at all. Thanks @dhilipkumars, I will revert and commit the change straight away.

Thanks all.